apps/cms/models/post.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from djrichtextfield.models import RichTextField
+
+from .base import AbstractContentType
+from ..fields import ColorField
+
+
+class Post(AbstractContentType):
+    """
+    Blog Post model based on "AbstractContentType".
+    Any "Custom Post Type" should be built on "AbstractContentType" base class.
+
+    AbstractContentType includes:
+     - title
+     - slug
+     - tags
+     - created_at
+     - updated_at
+    """
+    published = models.BooleanField(default=False)
+    published_on = models.DateTimeField(default=datetime.now, blank=True)
```

Comment:
can default just be `auto_now=True`? Also (and this is new to me too), I've been using `timezone.now()` from `django.utils` when working with timezones so it automatically gets timezone data from settings.

Reply:
Ah, so this is because it can be edited in the admin. Though, I think you're right that it should be `timezone.now`